tests/rptest/tests/usage_stats_test.py

```diff
@@ -0,0 +1,69 @@
+from rptest.clients.rpk import RpkTool
+from rptest.clients.types import TopicSpec
+from rptest.services.cluster import cluster
+from rptest.services.kgo_verifier_services import KgoVerifierProducer
+from rptest.tests.redpanda_test import RedpandaTest
+from ducktape.mark import matrix
+
+
+class UsageStatsReportingTest(RedpandaTest):
+    @cluster(num_nodes=4)
+    @matrix(replication_factor=[1, 3])
+    def test_usage_stats_reporting(self, replication_factor):
+        topic = TopicSpec(partition_count=1,
+                          replication_factor=replication_factor)
+        self.client().create_topic(topic)
+        stats = self.redpanda.usage_stats
+        # validate stats are initially 0
+        assert stats.batches_read == 0 and stats.batches_written == 0 and \
+            stats.bytes_read == 0 and stats.bytes_written == 0, \
+                "Before restarting Redpanda, the usage stats should be 0"
+
+        # restart nodes to trigger usage stats reporting
+        self.redpanda.restart_nodes(self.redpanda.nodes)
+        initial_stats = self.redpanda.usage_stats_dict.copy()
```

Comment:
nit:`RedpandaServiceABC.usage_stats_dict` is implemented using `dataclass.asdict`, which already makes a deep copy. So this invocation of `copy` is excessive. Either remove it or return just `self._usage_stats.__dict__` in `RedpandaServiceABC.usage_stats_dict`.